CMakeLists.txt

```diff
@@ -61,6 +61,14 @@ if(DEV_LOG_RAW_MESSAGE)
     add_definitions(-DDEV_LOG_RAW_MESSAGE)
 endif()
 
+# TODO(ale): enable i18n with add_definitions(-DLEATHERMAN_I18N) - PCP-257
+# TODO(ale): enable translation with set(LEATHERMAN_LOCALES "...;...") and
+# gettext_compile(${CMAKE_CURRENT_SOURCE_DIR}/locales share/locale)
+
+# Configure i18n
+file(GLOB_RECURSE PCP_CLIENT_SOURCES */src/*.cc */inc/*.hpp)
+gettext_templates(${CMAKE_CURRENT_SOURCE_DIR}/locales ${PCP_CLIENT_SOURCES})
+
```

Comment:
Enabling LEATHERMAN_I18N makes sense; ideally that should be a single thing you enabled. Since some platforms don't have sufficient Locale support, we'd have to guard it with LEATHERMAN_USE_LOCALES; there's probably a little cleanup that could be done around that, like providing a helper and cleaning up documentation to be a bit clearer.